wremnants/muon_efficiencies_smooth.py

```diff
@@ -112,7 +114,7 @@ def make_muon_efficiency_helpers_smooth(filename = data_dir + "/muonSF/allSmooth
             chargeTag = charge_tag if eff_type in chargeDependentSteps else "both"
             hist_name = f"SF_{nameTag}_{eratag}_{eff_type}_{chargeTag}"
             ## temporary patch for some missing SF histograms relevant only for dilepton (no warning needed otherwise)
-            if isoDefinition != "iso04vtxAgn":
+            if isoDefinition != "iso04vtxAgn" or era == "2018":
```

Comment:
This is ok, but maybe rather than checking era== 2018 you can have era != "2016PostVFP", since for 2017 there might be the same problem at some point.

Reply:
I recommend `if not (isoDefinition == "iso04vtxAgn" and era == "2016PostVFP")`